### PR TITLE
Feat/gdv 557 add header bar; adjust colors to coclico style

### PR DESCRIPTION
**Pull request by IoannaMi**
TODO: check with @danieltwigt the colors scheme

### CODE REVIEW

**Comment by danieltwigt** [comment]
Based on https://deploy-preview-8--coclico.netlify.app: 
Already looks good. Some comments/suggestions:
- Icon in upper left should be CoCliCo project icon (\src\assets\icons\icon-coclico.svg included in branch I created)
- Using white for the text in the CoCliCo blue header bar and side menu might be better for contrast. What's your opinion on this?

File: src/components/DataLayersCard.vue [hunk 1]
```diff
@@ -1,35 +1,52 @@
...
+              color="white100"
+              dark
+            > 
+              <v-row>
+                <v-col
+                  cols="1"
+                  class="ma-auto pa-0"
+                >
+                  <custom-icon
+                    name="cc"
```
Comment (CindyvdVries) [inline src/components/DataLayersCard.vue:36]: next ticket is making this generic?

**Review by CindyvdVries — APPROVED** [review]

File: src/components/AppHeader.vue [hunk 2]
```diff
@@ -1,11 +1,66 @@
...
+<script>
+  import {mapGetters} from 'vuex'  
+  export default { 
+    computed: {
+      ...mapGetters([ 'steppers' ])
+    }
+  }
+</script>
+<style>
+.stepper {
```
[the remark this answers is not in the document]
Comment (IoannaMi) [reply src/components/AppHeader.vue:55]: I just moved them to vuetify attributes. Good catch!

File: .eslintrc.js [hunk 3]
```diff
@@ -5,16 +5,22 @@ module.exports = {
...
   plugins: [ 'vue' ],
   rules: {
-    'eol-last': ["error", "always"],
+    'array-bracket-spacing': [ 'warn', 'always' ],
+    'eol-last': [ "error", "always" ],
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-unused-vars': 'warn',
+    'vue/script-indent': [ 'error', 2, { baseIndent: 1 } ],
```
[the remark this answers is not in the document]
Comment (IoannaMi) [reply .eslintrc.js:21]: It's for the script indent.

**Pull request merged**